source/jormungandr/tests/journey_common_tests.py

```diff
@@ -825,6 +825,16 @@ def test_call_kraken_boarding_alighting(self):
         assert journey['sections'][4]['departure_date_time'] == '20120614T233500'
         assert journey['sections'][4]['arrival_date_time'] == '20120614T233500'
 
+    def test_journey_from_non_valid_object_type(self):
+        """
+        When the departure is a non valid stop_area, the response status should be 404
```

Comment:
hum it's not only for stop_area, we can start from any places

so I guess it's more `When the departure is a not a Place, the response status should be 404`